bin/release.js

```diff
@@ -60,6 +61,82 @@ function checkEnv(varName) {
   }
 }
 
+/* We can change the release script to export a function instead.
+ * I already made this into a separate script so I think this is fine */
+async function uploadUpdateFiles(s3_key, s3_skey, new_version, app_dir, args) {
+  if (!args) args = [];
+
+  return new Promise((resolve, reject) => {
+    const submodule = cp.fork(
+      'bin/release-uploader.js',
+      [
+        '--access-key', s3_key,
+        '--secret-access-key', s3_skey,
+        '--version', new_version,
+        '--release-dir', app_dir,
+        ...args
```

Comment:
Yeah, I set some defaults to use a particular bucket since I was originally using the scripts directly from the CLI but I'm not sure I like that anymore (easy to mess up). I'm going to remove those defaults I think to remove that risk.